tasks/configure.yml

```diff
@@ -1,16 +1,16 @@
 ---
 
 - name: protect my.cnf
-  file: path='{{mysql_hardening_mysql_conf}}' mode=0600 owner=root group=root follow=yes
+  file: path='{{mysql_hardening_mysql_conf_file}}' mode=0600 owner=root group=root follow=yes
```

Comment:
Does the mysql unpriviliged process has to be able to read the config files? MySQL itself is started as root and then creates a child process running as user mysql.